### PR TITLE
Add position new

### PR DESCRIPTION
**Pull request by pratikshamandlik99**
New code with updated changes

### CODE REVIEW

File: ats-web/src/modules/Position/Position.js [hunk 1]
```diff
@@ -35,7 +51,12 @@ class Position extends React.Component {
       });
     };
     savePosition(position);
+   
   }
+  refreshPage() {
+    window.location.reload(false);
+  }
+
   refreshPage() {
```
Comment (aparna-salokhe) [inline ats-web/src/modules/Position/Position.js]: refreshPage() exists twice.. 
Why we are reloading page?? Can't we re-fetch the list of positions on submit?

File: ats-web/src/modules/Position/PositionModel.js [hunk 2]
```diff
@@ -9,12 +9,13 @@ export default class PositionModel {
     constructor() {
         this.id = 0;
         this.title = '';
-        this.experience = 0;
-        this.no_of_openings = 0;
-        this.project_id = null;
```
[the remark this answers is not in the document]
Comment (pratikshamandlik99) [reply ats-web/src/modules/Position/PositionModel.js]: I have kept project_id mandatory

File: ats-web/src/modules/Position/Position.js (hunk 1 above)
Comment (pratikshamandlik99) [reply ats-web/src/modules/Position/Position.js]: Resolved

File: ats-web/src/modules/Position/Position.js [hunk 3]
```diff
@@ -35,7 +51,12 @@ class Position extends React.Component {
       });
     };
     savePosition(position);
+   
   }
+  refreshPage() {
```
[the remark this answers is not in the document]
Comment (pratikshamandlik99) [reply ats-web/src/modules/Position/Position.js]: already removes duplicate one

File: ats-web/src/modules/Position/PositionModel.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (pratikshamandlik99) [reply ats-web/src/modules/Position/PositionModel.js]: Made project_id optional and kept null

File: ats-web/src/modules/Position/Position.js [hunk 4]
```diff
@@ -47,6 +68,18 @@ class Position extends React.Component {
     });
 
   }
+  
+
+  dropdown_onchange = (field, value) =>  {
```
[the remark this answers is not in the document]
Comment (pratikshamandlik99) [reply ats-web/src/modules/Position/Position.js]: Made common

**Pull request merged**